internal/template-bundle/bundle.go

```diff
@@ -47,6 +54,26 @@ func ReadBundle(filename string) (Bundle, error) {
 	}, nil
 }
 
+func RetrieveCommonTemplatesBundleFile(templateBundleDir string) (string, error) {
+	archDependentFileName := filepath.Join(templateBundleDir, commonTemplatesFilePrefix+runtime.GOARCH+"-"+common_templates.Version+commonTemplatesFileSuffix)
+	archIndependentFileName := filepath.Join(templateBundleDir, commonTemplatesFilePrefix+common_templates.Version+commonTemplatesFileSuffix)
```

Comment:
The string concatenation is a little confusing here. Can you use `fmt.Sprintf()`? The format string could be:
```
common-templates-%s-%s.yaml`
```

Reply:
Done